src/components/TodoError.tsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
You should avoid using eslint-disable unless it's absolutely necessary. If the warning is about 'jsx-a11y/control-has-associated-label', it's better to fix it by providing a label for the control.